src/components/List/List.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bem from 'bem';
+import styles from './List.scss';
+
+const { block } = bem({
+    name: 'List',
+    classnames: styles,
+    propsToMods: ['isDividedList', 'Component']
+});
+
+const List = props => {
+    const { children, Component, isDividedList, ...rest } = props;
+    return (
+        <Component {...rest} {...block(props)}>
+            {React.Children.map(children, child => React.cloneElement(child, { isDividedList }))}
+        </Component>
+    );
+};
+
+List.displayName = 'List';
+
+List.propTypes = {
+    /** List Items */
+    children: PropTypes.node,
+    /** The HTML or React component to be used to render the list  */
+    Component: PropTypes.string,
```

Comment:
I'm not sure if we really need to make the list type configurable, I'd probably just go for `ul` out of the box. This makes it more complicated, especially considering class naming / bem. But if you really want to keep it, I would at least predefine the types: `oneOf(['ul', 'ol', ...])`.

Reply:
I don't see this as a problem to have it. `ul` and `ol` are standard html tags that one expects to have for a list. It is already implemented, I rather not spend time on removing it for no real reason.
I fixed the prop types as suggested.

Reply:
The reason for not having it is that there is no necessity and because we don't have a requirement we are implementing this on the basis of an assumption and it makes the component more complicated. Hence my comment.

Reply:
I don't see it is as obstacle and it was part of the original design

Reply:
I don't get why you're digging in your heels to keep this but I'm not going to continue challenging it either. Not saying it's an obstacle besides it makes the code more complex while we probably won't even need it.